oaknational/Oak-Web-Application
# Remove old right margin, left over from the wobbly border visualiser

## Description
In our migration over from the wobbly border curriculum visualiser. We left a right margin on the swimming info box.

## Issue(s)

Fixes `CUR-1051`

## How to test

1. Go to https://deploy-preview-3015--oak-web-application.netlify.thenational.academy
2. Go to PE/primary in the visualiser
3. Check swimming all-years doesn't have odd right margin

## Screenshots

How it used to look (delete if n/a):
<img width="471" alt="Screenshot 2024-11-21 at 11 42 53" src="https://github.com/user-attachments/assets/eee8a380-dbee-4a51-a689-fe305969fdd6">

How it should now look:
<img width="471" alt="Screenshot 2024-11-21 at 11 42 43" src="https://github.com/user-attachments/assets/6a747f69-807b-4021-bfbd-6b4f114d4d7f">
